resources/packs/vcd/vcd.lr

```diff
@@ -0,0 +1,279 @@
+option go_package = "go.mondoo.com/cnquery/resources/packs/vcd"
+
+// VMware Cloud Director
+vcd {
+  // vCD Org resources
+  organizations() []vcd.organization
+  // vCD Provider VDC
+  providerVDCs() []vcd.vdcProvider
+  // vCD Network Pools
+  networkPools() []vcd.networkPool
+  // vCD External Networks
+  externalNetworks() []vcd.externalNetwork
+  // vCenter server attached to VCD
+  serverInstances() []vcd.serverInstance
+}
+
+// VMware Cloud Director Org resource
+vcd.organization @defaults("name") {
+  id string
+  // Org name
+  name string
+  // Org full name
+  fullName string
+  // Indicates if this organization is enabled
+  isEnabled bool
+  // Org description
+  description string
+  // Virtual Machines
+  vms() []vcd.vm
+  // Rights
+  rights() []vcd.right
+  // Virtual Data Centers (VDC)
+  vdcs() []vcd.vdc
+  // VDC groups
+  vdcGroups() []vdc.vdcGroup
```

Comment:
"vcd.vdcGroup"?

Reply:
Yes, your suggestion is correct

vcd - VMware Cloud Director
vdc - Virtual Data Center

Do not ask me why VMware uses both 3 letter acronyms at the same time :-)